docs/hub/spaces-zerogpu.md

```diff
@@ -88,6 +88,11 @@ This sets the maximum function runtime to 120 seconds. Specifying shorter durati
 
 By leveraging ZeroGPU, developers can create more efficient and scalable Spaces, maximizing GPU utilization while minimizing costs.
 
+## Recommendations
+
+If your demo uses a large model, we recommend [these optimizations](https://huggingface.co/blog/zerogpu-aoti) to maximize the advantages of ZeroGPU hours and
```

Comment:
it might be better to spell out the optimisations here - maybe you can add them as examples

Reply:
Sure. Is it fine to include them in this doc or did you want me to create a separate doc?

Reply:
I meant more just add the keywords in the same line `we recommend using X, Y and Z optimisations, you can read about them here. These will help maximize your ZeroGPU usage`

(or something along this line)